templates/openstack/what-is-openstack.html

```diff
@@ -2,206 +2,593 @@
       </p>
     </div>
   </div>
 </section>
 
 <section class="p-strip is-bordered">
-  <div class="u-fixed-width">
-    {{
-      image(
-        url="https://assets.ubuntu.com/v1/2d2a0b2a-what+is+open+stack+diagram.svg",
-        alt="What is OpenStack?",
-        height="250",
-        width="850",
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>OpenStack ensures maximum multi-cloud cost optimisation</h2>
+      <p>A private cloud is a cost-efficient extension of the public cloud infrastructure. Since more and more organisations are deciding to use <a href="/cloud">multi-cloud architecture</a>, implementing a private cloud is a natural step once the number of workloads grows. Although CapEx costs associated with an initial deployment of the private cloud are high, its OpEx costs are significantly lower compared to public clouds. As a result, the aggregated total cost of ownership (TCO) per virtual machine (VM) is lower when running workloads in the long term and at scale. This allows businesses to optimise their infrastructure costs and always run their applications where it makes most of the sense from the economical point of view.</p>
+      <p>
+        <a href="/tco-calculator">Learn how much you can save with OpenStack&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-4 u-align--center u-vertically-center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/dcb2963c-openstack+cloud+outlines.svg",
+        alt="",
+        width="263",
+        height="150",
         hi_def=True,
-        loading="auto",
-      ) | safe
-    }}
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
   </div>
 </section>
 
 <section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-9">
+      <h2>Is OpenStack just a virtualisation management platform?</h2>
+      <p>Unlike traditional virtualisation management platforms, such as VMware vSphere or Red Hat Virtualization Manager, OpenStack is a fully functional cloud platform as defined by the <a class="p-link--external" href="https://www.nist.gov/programs-projects/nist-cloud-computing-program-nccp">National Institute of Standards and Technology</a>. This means that OpenStack basically resembles the behaviour of public clouds. Users can request compute, network and storage resources on-demand through a self-service portal. Although they are provided in the form of virtual machines, virtual network and virtual disks, exactly as in the traditional virtualisation management platforms, they are defined through the APIs. For businesses, the ability to define virtualised resources programmatically enables fast-paced infrastructure automation and cloud-style operations.</p>
+      <p>But there are more differences between OpenStack and traditional virtualisation management platforms. Refer to the following table for more information:</p>
+    </div>
+    <div class="col-12">
```

Comment:
this `col-12` needs to be in a new row, as the row it's currently in is already populated with a `col-9`